components/ActivityCardPreview/index.js

```diff
@@ -1,21 +1,16 @@
 import Image from "next/image";
-import styled from "styled-components";
 import getFilteredCategories from "@/utils/filterCategories";
 import Link from "next/link";
 import BookmarkButton from "../BookmarkButton";
 import { StyledCategory, StyledCateogriesUl } from "../StyledList";
 
-const StyledImage = styled(Image)`
-  width: auto;
-  height: auto;
-`;
-
 export default function ActivityCardPreview({
   activity,
   isBookmarked,
```

Comment:
as mentioned in another comment the isBookmarked prop is not necessary here (it is already a part of 'activity').